docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    cand_index_left = _find_candidates(T, m, M_T, Σ_T, min_dist, is_cands_init, direction='right')
</pre><p>Shouldn't this be <code>direction='left'</code> ??</p><p> </p><p>When or why would you ever do it this way? It would be slower, right? Since you aren't using the "right" candidates to eliminate the "left" candidates?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
> Shouldn't this be `direction='left'`

I probably changed it by mistake. You are correct. That is `left`.


> When or why would you ever do it this way? It would be slower, right? Since you aren't using the "right" candidates to eliminate the "left" candidates?

Yes, you are correct! It is definitely slower. My purpose was to show it here so if we have two computing resources, we can ask each to take care of one direction. However, now I think we can use them to speed up calculation in one direction and then use them again for the other direction 😆 

Sorry for this extra cell! I will remove it.

Reply:
<p>Sonds good</p>